TACC/Core-Styles
# feat(elements): fake table using paragraphs

## Overview

Allow a single-column table to built from paragraphs.

## Related

None.

## Changes

- feat(css): `.pargagraph-table`
- feat(demo): `table--fake`

## Testing

1. Open http://localhost:3000/components/detail/table--fake.
2. Verify paragraph-based table matches real table with no thead.*
3. Open http://localhost:3000/components/detail/s-paragraph-table.
4. Verify table matches the one from step 1.

<sup>* Ignore the right border on the real table. That is a bug with real single-column tables.</sup>

## UI

| trumps | elements |
| - | - |
| ![trumps](https://user-images.githubusercontent.com/62723358/218855141-1b6fb8da-f891-45f0-8e8f-165d5e46e399.png) | ![elements](https://user-images.githubusercontent.com/62723358/218855137-45349730-70fa-41ef-aba1-5375a73554b9.png) |
